tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+class ActionSelenium(argparse.Action):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    def __call__(self, parser, namespace, values, option_string=None):
+        browsers = values.split(',')
+        for browser in browsers:
+            try:
+                # Attempt to create class instance to see if everything is fine.
+                import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)
+            except Exception:
```

Comment:
I think instantiating the browser to check if it works properly is also a part of the "validation" process. I don't see much gain in just checking that provided browsers are "importable" from selenium library.
It will be better to keep that part of the error message and trying to instantiate the browsers here instead. Because there are other modules like `remote`, `support` in selenium which would get imported but do not have webdrivers - so these will give `WebDriverException` afterwards during the tests. Also if someone doesn't have the required drivers installed for a particular browser, it will again throw a similar Exception. So the `import_string` is just half of the validation.